### PR TITLE
Python's built-in rational in the documentation; integer division in Py 2; tuple behavior

### PR DESCRIPTION
**Pull request by TrangOul**
Some rather minor documentation fixes and improvements.

#### References to other Issues or PRs
Recreated as https://github.com/sympy/sympy/pull/26815; please review the newer one.

#### Brief description of what is fixed or changed
Added a mention about `fractions.Fraction` in the tutorial.
Fixed typos in "sympify".
Added some details about tuples in Python.
Boolean params as `bool`s, not `int`s.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: doc/src/explanation/gotchas.rst [hunk 1]
```diff
@@ -718,6 +719,18 @@ square brackets.
     >>> (x)
     x
 
+    Actually parentheses are not needed for non-empty tuples; the commas are:
```
Comment (smichr) [inline doc/src/explanation/gotchas.rst:722]:
```suggestion
    Parentheses are not needed for non-empty tuples; the commas are:
```

File: doc/src/explanation/gotchas.rst [hunk 2]
```diff
@@ -364,7 +365,7 @@ you don't have to worry about this problem:
     >>> print(solve(7*x -22, x))
     [22/7]
     >>> 22/7  # If we just copy and paste we get int 3 or a float
```
Comment (smichr) [inline doc/src/explanation/gotchas.rst:367]:
```suggestion
    >>> 22/7  #copy and paste gives int 3 (in Python 2) or a float (in Python 3)
```

Comment (smichr) [reply doc/src/explanation/gotchas.rst:367]: And then the additions to the following line are not needed.

File: doc/src/explanation/gotchas.rst [hunk 3]
```diff
@@ -332,11 +332,12 @@ automatically, but there is one gotcha you should be aware of.  If you
 do ``<number>/<number>`` inside of a SymPy expression, Python will
 evaluate the two numbers before SymPy has a chance to get
 to them.  The solution is to :func:`~.sympify` one of the numbers, or use
-:obj:`~.Rational`.
+:obj:`~.Rational` (or alternatively Python's `Fraction
```
Comment (smichr) [inline doc/src/explanation/gotchas.rst:335]:
```suggestion
:obj:`~.Rational` (or Python's `Fraction
```

File: doc/src/explanation/gotchas.rst [hunk 4]
```diff
@@ -377,7 +378,7 @@ you don't have to worry about this problem:
     >>> S("22/7")
     22/7
 
-Also, if you do not use :command:`isympy`, you could use ``from
+Also, if you use Python 2 and do not use :command:`isympy`, you could use ``from
```
Comment (asmeurer) [inline doc/src/explanation/gotchas.rst:381]: We should just remove all mentions of this. No one uses Python 2 any more.

File: doc/src/explanation/gotchas.rst (hunk 3 above)
Comment (asmeurer) [reply doc/src/explanation/gotchas.rst:335]: While this is true I don't think this is going to be the easiest solution for most people. If you are going to construct a class, you should just construct the SymPy class.

File: doc/src/explanation/gotchas.rst (hunk 4 above)
Comment (TrangOul) [reply doc/src/explanation/gotchas.rst:381]: I agree with that, but IMO it deserves a separate PR, though.
Anyway, [the documentation](https://docs.sympy.org/latest/install.html) says:
> SymPy officially supports Python 3.8, 3.9, 3.10, and PyPy.

File: doc/src/explanation/gotchas.rst (hunk 3 above)
Comment (TrangOul) [reply doc/src/explanation/gotchas.rst:335]: While using `sympy.Rational(fractions.Fraction(x, y))` is an overkill and I don't expect anyone to do so (unless that `Fraction` is a result of another calculation), I think it's good to make the users aware that Python also has a rational class with unlimited precision, not just `float`.

**Comment by TrangOul** [comment]
Please review https://github.com/sympy/sympy/pull/26815 instead. It's the same, but I had to rewrite git history, and to make everything clean, I've made another branch.

**Pull request closed without merging**